coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -174,28 +174,32 @@ def _extract_doc_comment(content, line, column, markers):
 
 def _compile_multi_match_regex(strings):
     """
-    Compiles a regex object that matches each of the given strings.
+    Compiles a regex object that checks for indentation before the starting
+    marker (so as to ignore triple quote string literals) and group matches
+    each of the given strings.
 
     :param strings: The strings to match.
     :return:        A regex object.
     """
-    return re.compile('|'.join(re.escape(s) for s in strings))
+    return re.compile('^\s*(?P<marker>' +
+                      ('|'.join(re.escape(s) for s in strings)) +
+                      ')')
 
 
 def _extract_doc_comment_from_line(content, line, column, regex,
                                    marker_dict, docstyle_definition):
     cur_line = content[line]
     begin_match = regex.search(cur_line, column)
     if begin_match:
-        indent = cur_line[:begin_match.start()]
         column = begin_match.end()
-        for marker in marker_dict[begin_match.group()]:
+        indent = cur_line[:column-len(begin_match.group('marker'))]
```

Comment:
yup i like the idea of group matching `indent` too and use it here to extract indent :+1: